scripts/lite/shelley-testnet.sh

```diff
@@ -40,27 +62,31 @@ for i in 1 2 3; do
   # - KES signing key
   # - VRF signing key
   # - Operational certificate
+  # VRF keys have already been generated in the shelley genesis create command
 
-  # Generate VRF keys
-  mkdir -p "${data_dir}/genesis/vrf-keys"
-  cardano-cli shelley node key-gen-VRF \
-      --verification-key-file "${data_dir}/genesis/vrf-keys/vrf-$i.vkey" \
-      --signing-key-file      "${data_dir}/genesis/vrf-keys/vrf-$i.skey"
 
   # Generate a KES keys
-  mkdir -p "${data_dir}/genesis/kes-keys"
+  mkdir -p "${data_dir}/node-$i"
   cardano-cli shelley node key-gen-KES \
-    --verification-key-file "${data_dir}/genesis/kes-keys/kes-$i.vkey" \
-    --signing-key-file      "${data_dir}/genesis/kes-keys/kes-$i.skey"
+    --verification-key-file "${data_dir}/node-$i/kes.vkey" \
+    --signing-key-file      "${data_dir}/node-$i/kes.skey"
+
+  # Move genesis delegate keys generated in shelley genesis create command to it's
+  # respective node folder.
+
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.skey"      "${data_dir}/node-$i/hotkey.skey"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.vkey"      "${data_dir}/node-$i/hotkey.vkey"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.counter"   "${data_dir}/node-$i/counterFile"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.vrf.skey"  "${data_dir}/node-$i/vrf.skey"
+  mv "${data_dir}/genesis/delegate-keys/delegate$i.vrf.vkey"  "${data_dir}/node-$i/vrf.vkey"
 
   # Issue an operational certificate:
-  mkdir -p "${data_dir}/genesis/delegate-opcerts"
   cardano-cli shelley node issue-op-cert \
       --kes-period 0 \
-      --kes-verification-key-file                  "${data_dir}/genesis/kes-keys/kes-$i.vkey"  \
-      --cold-signing-key-file                      "${data_dir}/genesis/delegate-keys/delegate$i.skey" \
-      --operational-certificate-issue-counter-file "${data_dir}/genesis/delegate-keys/delegate$i.counter" \
-      --out-file                                   "${data_dir}/genesis/delegate-opcerts/delegate$i.opcert"
+      --kes-verification-key-file                  "${data_dir}/node-$i/kes.vkey"  \
+      --cold-signing-key-file                      "${data_dir}/node-$i/hotkey.skey" \
+      --operational-certificate-issue-counter-file "${data_dir}/node-$i/counterFile" \
+      --out-file                                   "${data_dir}/node-$i/opcert"
 
```

Comment:
I would put the `cabal run exe:cardano-node` in a separate loop to cleanly separate the set up and the running of the node and move the `rmdir "${data_dir}/genesis/delegate-keys"` to in between the loops.